atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -505,10 +540,11 @@ public Void apply(Cassandra.Client client) throws Exception {
         }
     };
 
-    final FunctionCheckedException<Cassandra.Client, Void, Exception> createInternalMetadataTable = new FunctionCheckedException<Cassandra.Client, Void, Exception>() {
+    final FunctionCheckedException<Cassandra.Client, Void, Exception> createInternalTables = new FunctionCheckedException<Cassandra.Client, Void, Exception>() {
         @Override
         public Void apply(Cassandra.Client client) throws Exception {
             createTableInternal(client, CassandraConstants.METADATA_TABLE);
```

Comment:
Are we concerned about the race effecting the metadata table?  Seems like only the lock table can be said to have the quality that it is alright if it suffers from the table creation race since it is no longer valid after a restart.